kuma/core/tests/test_checks.py

```diff
@@ -0,0 +1,70 @@
+import json
+import os
+
+from django.core.checks import Error, Warning
+
+from kuma.core import checks
+
+
+def test_react_i18n_check(tmpdir, settings):
+    settings.ACCEPTED_LOCALES = ['sv-SE', 'en-US']
+    settings.BASE_DIR = base_dir = str(tmpdir)
+    # tmpdir.mkdir('static')
+    os.makedirs(os.path.join(base_dir, 'static', 'jsi18n', 'sv-SE'))
+    os.makedirs(os.path.join(base_dir, 'static', 'jsi18n', 'en-US'))
+
+    # Because we'll reuse this a lot
+    general_hint = "Run 'make compile-react-i18n'"
+
+    # Missing the react.json files
+    errors = checks.react_i18n_check(None)
+    assert len(errors) == 2
+    sv_path = os.path.join(base_dir, 'static', 'jsi18n', 'sv-SE', 'react.json')
+    assert errors[0] == Error(
+        'Locale file {} does not exist'.format(sv_path),
+        hint=general_hint,
+        id=checks.ERROR_MISSING_I18N_FILE
+    )
+    # Create both but make one of them weird
+    en_path = os.path.join(base_dir, 'static', 'jsi18n', 'en-US', 'react.json')
+    with open(en_path, 'w') as f:
+        json.dump({
+            "formats": [],
+            "catalog": {},
+            "plural": "multiple"
+        }, f)
+    with open(sv_path, 'w') as f:
+        f.write('{{not valid JSON')
+    errors = checks.react_i18n_check(None)
+    assert len(errors) == 1
+    json_decode_error = 'Expecting property name: line 1 column 2 (char 1)'
```

Comment:
nit: seems a little brittle to make your test depend on a error message that is out of your control. Maybe just check the hint and the id?

Reply:
I agree. I guess I could use the `mock.ANY` or whatever it's called.

Reply:
Scratch that, it's a string inside a string. I'll think of something else.